CMakeLists.txt

```
@@ -10,7 +10,12 @@ include( StlabUtil )
 find_package( Boost 1.74.0 OPTIONAL_COMPONENTS unit_test_framework )
 find_package( libdispatch )
 find_package( Threads )
-find_package( Qt6 COMPONENTS Core )
+
+# Search Qt 6 and Qt 5
+# Since QT_DEFAULT_MAJOR_VERSION is not set, Qt::Core, qt_xxx will refer to Qt6 as it is found first
+# (see https://doc.qt.io/qt-6/cmake-qt5-and-qt6-compatibility.html)
+find_package( Qt6 QUIET COMPONENTS Core )
+find_package( Qt5 QUIET COMPONENTS Core )
```

```suggestion
find_package( Qt5 QUIET COMPONENTS Core )
find_package( Qt6 QUIET COMPONENTS Core )
```

I suggest

1. Remove the comment since it is repeating documentation and isn't relevant to our code since we're using QtN prefixes everywhere now.
2. Put this in Qt5 and then Qt6 order to maintain alphabetical ordering of these `find_package` calls.

---

Done, I also sorted the `find_dependency`s in `stlabConfig.cmake.in` to facilitate checking them against the matching `find_package`.